duckduckgo/macos-browser
# Update autoconsent to 5.1.0

Task/Issue URL: https://app.asana.com/0/0/1205171013442986/f
Tech Design URL:
CC:

**Description**:
Upgrading autoconsent. This has a few fixes to address breakage

**Steps to test this PR**:
1. Run the browser, use the following config: https://jsonblob.com/1135512371392798720
2. Verify that cookie pop-ups are correctly managed on
  - https://www.buhl.de/mein-buhlkonto/
  - https://news.sky.com/



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
